packages/app/docs/stories/icons/rainbowLogo.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const RainbowLogo = (props) => {
```

Comment:
This could be a candidate for an icon under @rainbow-modules/icons since it's going to be used everywhere, another PR off course but I will let @TahimiLeonBravo to make that call